renku/domain_model/project.py

```diff
@@ -115,20 +131,24 @@ def from_client(
         if name is None:
             raise ValueError("Project 'name' not set and could not be generated")
 
+        if namespace is None:
+            raise ValueError("Project 'namespace' not set and could not be generated")
```

Comment:
I know that this was in the code before but I don't see anywhere if we catch `ValueError`s. Should we use a Renku exception instead? It also applies to other places in this method.